pyani_plus/workflows/snakemake_branchwater.smk

```diff
@@ -54,16 +56,17 @@ rule manysearch:
         db=config["db"],
         run_id=config["run_id"],
         outdir=config["outdir"],
+        temp=config["temp"],
         extra=config["extra"],  # This will consist of either `scaled=X` or `num=X`.
     input:
         expand("{{outdir}}/{genome}.sig", genome=sorted(indir_files)),
     output:
         "{outdir}/manysearch.csv",
     shell:
         """
-        sourmash sig collect --quiet -F csv -o all_sigs.csv {input} > {output}.log 2>&1 &&
-        sourmash scripts manysearch -m DNA --quiet -t 0 -o {output} all_sigs.csv all_sigs.csv >> {output}.log 2>&1 &&
+        sourmash sig collect --quiet -F csv -o all_sigs.csv {input:q} > "{output}.log" 2>&1 &&
```

Comment:
Note the use of ``{input:q}`` to do the quoting (handles multiple files) rather than ``"{input}"`` (would lump multiple files together).